### ISSUE TITLE
[ingester] Crash if Kafka is specified as storage

### ISSUE THREAD

**Issue by yurishkuro** [jaegertracing/jaeger#1190]
Reported by @ledor473 - https://github.com/jaegertracing/jaeger/pull/1094#issuecomment-439221399

I've just realized that if you run the Ingester with `SPAN_STORAGE_TYPE=elasticsearch,kafka`, the Ingester crash with the following:
```
WARNING: multiple span storage types have been specified. Only the first type (elasticsearch) will be used for reading and archiving.

flag redefined: kafka.brokers
panic: flag redefined: kafka.brokers

goroutine 1 [running]:
flag.(*FlagSet).Var(0xc000080d80, 0xefd9a0, 0xc00018d080, 0xe04f6e, 0xd, 0xe2cb58, 0x4b)
	/home/travis/.gimme/versions/go1.11.1.linux.amd64/src/flag/flag.go:805 +0x529
flag.(*FlagSet).StringVar(0xc000080d80, 0xc00018d080, 0xe04f6e, 0xd, 0xe05426, 0xe, 0xe2cb58, 0x4b)
	/home/travis/.gimme/versions/go1.11.1.linux.amd64/src/flag/flag.go:708 +0x8a
flag.(*FlagSet).String(0xc000080d80, 0xe04f6e, 0xd, 0xe05426, 0xe, 0xe2cb58, 0x4b, 0xe00e66)
	/home/travis/.gimme/versions/go1.11.1.linux.amd64/src/flag/flag.go:721 +0x8b
github.com/jaegertracing/jaeger/cmd/ingester/app.AddFlags(0xc000080d80)
	/home/travis/gopath/src/github.com/jaegertracing/jaeger/cmd/ingester/app/flags.go:84 +0x6e
github.com/jaegertracing/jaeger/pkg/config.AddFlags(0xc00015a0f0, 0xc000186240, 0xc000159f28, 0x5, 0x5, 0xc00003a042, 0xd)
	/home/travis/gopath/src/github.com/jaegertracing/jaeger/pkg/config/config.go:36 +0x56
main.main()
	/home/travis/gopath/src/github.com/jaegertracing/jaeger/cmd/ingester/main.go:138 +0x59f
```

I'm not sure how, but I feel we should add a validation on the storage type to prevent the use to `KAFKA` in the Ingester. Unless we want to support Kafka -> Kafka with the Ingester, but that feels wrong

**Comment by yurishkuro** [comment]
yeah, we haven't envisioned kafka->kafka

**Comment by ledor473** [comment]
I’ll try to do the fix tomorrow, I think there’s a 1.8.1 that is planned and this fixe could be part of it
